src/mol-canvas3d/canvas3d.ts

```diff
@@ -865,7 +868,7 @@ namespace Canvas3D {
         }
 
         const contextLostSub = contextLost?.subscribe(() => {
-            pause(true);
+            isContextLost = true;
```

Comment:
@arose  we need to use this (or a similar approach that doesn't reset the animation loop), because the plugin animation is managed by the `PluginAnimationLoop`